### PR TITLE
Fixes #206. Added changes to create profile as per refactored API

### LINKED ISSUE #206: Support refactored OpenSDS Hotpot  Provisioning APIs
The Hotpot APIs are being refactored and updated from OpenAPI spec 2.0 to 3.0.  
Alongwith that there are a few parameter changes to make the APIs more efficient and provide more control. 
These changes will have an impact on the Dashboard and how some of the fields are used (or not used) from the API responses.   
The first set of changes is in the Provisioning APIs:  
- Profiles  
- Pools 
- Volumes
Reference Link: [Click here](https://docs.google.com/document/d/1huHIIZQbjbnw1qSFmUsi0ER_vBuVbtDjHNpMcd9h9Ho/edit#heading=h.rxhmi6xlbhyh) to view the design spec.

### PR DESCRIPTION
**Pull request by anvithks**
Added the new fields as per the refactored Hotpot API to Create Profile.  
- minIOPS  
- minBWS  
- latency  

Screenshot below shows the changes in the Create Profile Page.  
206-hotpot-create-profile

### CODE REVIEW

**Comment by anvithks** [comment]
@PravinRanjan10 @wisererik Please review the PR

File: src/app/business/profile/createProfile/createProfile.component.ts [hunk 1]
```diff
@@ -229,7 +229,10 @@ export class CreateProfileComponent implements OnInit {
             isExisted:this.I18N.keyID['sds_isExisted'],
         },
         "maxIOPS": { required: this.I18N.keyID['sds_required'].replace("{0}","MaxIOPS")},
+        "minIOPS": { required: this.I18N.keyID['sds_required'].replace("{0}","MinIOPS")},
```
Comment (kumarashit) [inline src/app/business/profile/createProfile/createProfile.component.ts]: What it takes to say it as "Maximum IOPS", "Minimum IOPS" etc? any specific backend support?

Comment (anvithks) [reply src/app/business/profile/createProfile/createProfile.component.ts]: Probably a coffee or two :P 
Can be done.

File: src/app/business/profile/createProfile/createProfile.component.html [hunk 2]
```diff
@@ -43,18 +43,42 @@ <h1>{{I18N.keyID['sds_profile_create_title']}}</h1>
...
+                            </form-item>
+                            <form-item label="{{label.latency}}" [required]="true">
+                                <div class="input-text-vertical-align">
+                                    <input pInputText type="text" formControlName="latency" />
+                                    <span>{{I18N.keyID['sds_profile_latency_unit']}}</span>
+                                </div>
+                            </form-item>
+                        </div>
+                        <div class="ui-grid-col-6">
+                            <form-item label="{{label.minIOPS}}" [required]="true">
```
Comment (PravinRanjan10) [inline src/app/business/profile/createProfile/createProfile.component.html]: Please set the new parameters (MinIOPS, MinBWS and latency) as optional. Not as required parameters.

Comment (anvithks) [reply src/app/business/profile/createProfile/createProfile.component.html]: OK.

**Comment by anvithks** [comment]
@PravinRanjan10 @kumarashit please check.

**Comment by anvithks** [comment]
This PR is tested with a local backend. Integration test with the updated API still needs to be performed after the updated APIs are available in the development branch of hotpot.
Depends on [opensds/opensds #1074](https://github.com/opensds/opensds/pull/1074)

**Comment by anvithks** [comment]
@PravinRanjan10 @kumarashit @wisererik @skdwriting Please review, approve and merge.

**Comment by PravinRanjan10** [comment]
LGTM

**Comment by yuejuSDS** [comment]
LGTM

**Pull request merged**